db/db.js

```diff
@@ -14,6 +14,97 @@ const connect = async () => {
   }
 }
 connect();
+// creating the tables
+const User = sequelize.define('User', {
+  userName: Sequelize.STRING,
+  city: Sequelize.STRING,
+  desciption: Sequelize.STRING,
+  googleId: Sequelize.STRING,
+  gmail: Sequelize.STRING,
+  cell: Sequelize.TEXT,
+});
+
+// const Messages = sequelize.define('Messages', {
```

Comment:
include 'TODO:' tag with a description to reference code commented out for future use